ecc/bls12381/g1.go

```diff
@@ -52,14 +54,17 @@ func (g *G1) Neg() { g.y.Neg() }
 // SetIdentity assigns g to the identity element.
 func (g *G1) SetIdentity() { g.x = ff.Fp{}; g.y.SetOne(); g.z = ff.Fp{} }
 
+// isZeroProj returns true if the point is not a projective point.
```

Comment:
Not the greatest of names imho: sounds like it checks if the point is the identity of the curve in projective representation. How about "isNotP1Point"?

Reply:
changed to `isValidProjective`.
Interestingly, it would be good to avoid this case by-design.